.github/workflows/deploy.yml

```
@@ -0,0 +1,115 @@
+        run: |
+          docker build -t "$REPO_URI:$IMAGE_TAG" -t "$REPO_URI:latest" .
+
+      # 4) ECR 푸시
+      - name: Docker push
+        run: |
+          docker push "$REPO_URI:$IMAGE_TAG"
+          docker push "$REPO_URI:latest"
+
+      # 5) ECS 서비스 업데이트를 위한 AWS CLI 재설정: AWS_REGION 을 northeast-2로 재설정해야되기 때문
+      - name: Configure AWS Credentials (ECS 리전)
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.12)

[error] the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)